web/themes/custom/pineapple/assets/js/menu.behavior.js

```diff
@@ -0,0 +1,15 @@
+(function ($, Drupal, drupalSettings) {
+
+  'use strict';
+
+  Drupal.behaviors.menu = {
+    attach: function (context, settings) {
+      $('.menu .hamburger').once('menu-functionality').each(function() {
```

Comment:
You don't need once or each here, once is causing js errors see in incognito window

Reply:
I will need it when I run the slide toggle function. But I see the js errors. Noted.

Reply:
done